src/Compilers/Core/MSBuildTask/Microsoft.CSharp.Core.targets

```diff
@@ -50,8 +50,9 @@
     </PropertyGroup>
 
     <PropertyGroup>
+      <MaxLangVersion Condition="'$(MaxLangVersion)' == ''">7.3</MaxLangVersion>
```

Comment:
https://github.com/dotnet/project-system/pull/5028 is defining a property "MaxSupportedLangVersion" -- is that what you intended to put here?

Reply:
This property in general doesn't seem very good. It's the `MaxLangVersion`, but this is only true of .net framework apps and older .net core apps? This doesn't seem correct.

Reply:
Yeah, MaxSupportedLangVersion is what I meant. I will update.